get rid of -fvia-C (fixes build against binutils-2.21)

Fixes build failure reported by Hans-Peter Deifel:

> [1 of 2] Compiling System.INotify.Masks ( dist/build/System/INotify/Masks.hs, dist/build/System/INotify/Masks.o )
> temp/ghc8640_0/ghc8640_0.s: Assembler messages:
> temp/ghc8640_0/ghc8640_0.s: Error: .size expression for s1bK_ret does not evaluate to a constant
> temp/ghc8640_0/ghc8640_0.s: Error: .size expression for s1bT_ret does not evaluate to a constant
> temp/ghc8640_0/ghc8640_0.s: Error: .size expression for s1bR_ret does not evaluate to a constant

Bug-URL: https://bugs.gentoo.org/show_bug.cgi?id=380967
diff --git a/hinotify.cabal b/hinotify.cabal
index 388d920..dd4095e 100644
--- a/hinotify.cabal
+++ b/hinotify.cabal
@@ -33,6 +33,6 @@ library
     other-modules:
         System.INotify.Masks
 
-    ghc-options:    -fvia-C -Wall
+    ghc-options:    -Wall
 
     hs-source-dirs: src
